pyccl/core.py

```diff
@@ -164,45 +164,231 @@ class Parameters(object):
               for details.
 
     Args:
-        Omega_c (float): Cold dark matter density fraction.
-        Omega_b (float): Baryonic matter density fraction.
-        h (float): Hubble constant divided by 100 km/s/Mpc; unitless.
-        A_s (float): Power spectrum normalization. Exactly one of A_s
-                     and sigma_8 is required.
-        n_s (float): Primordial scalar perturbation spectral index.
-        Omega_k (float, optional): Curvature density fraction. Defaults to 0.
-        N_nu_rel (float, optional): Effective number of massless neutrinos
-                                    present. Defaults to 3.046.
-        N_nu_mass (float, optional): Number of massive neutrinos present.
-                                     Defaults to 0.
-        m_nu (float, optional): total mass in eV of the massive neutrinos
-                                present. Defaults to 0.
+        Omega_c (:obj:`float`): Cold dark matter density fraction.
+        Omega_b (:obj:`float`): Baryonic matter density fraction.
+        h (:obj:`float`): Hubble constant divided by 100 km/s/Mpc; unitless.
+        A_s (:obj:`float`): Power spectrum normalization. Exactly one of A_s
+            and sigma_8 is required.
+        sigma8 (:obj:`float`): Variance of matter density perturbations at
+            an 8 Mpc/h scale. Exactly one of A_s and sigma_8 is required.
+        n_s (:obj:`float`): Primordial scalar perturbation spectral index.
+        Omega_k (:obj:`float`, optional): Curvature density fraction.
+            Defaults to 0.
+        Omega_g (:obj:`float`, optional): Density in relativistic species
+            except massless neutrinos. The default of `None` corresponds
```

Comment:
In general, we should edit the CCL note in doc/0000_ccl_note/main.tex to cover these changes. But specifically this warning should be given there. Perhaps we could check for inconsistency and give a warning in the code itself?

Reply:
Just to make sure I understand, you want me to warn the user if Omega_g is zero but T_CMB is not? This is a very common use case for late-time LSS computations. I am a bit worried this would be too noisy for an option people only opt into. 

However, if you still want to go ahead with the warning I am happy to do so!

Reply:
Sounds good. Do you mean the DESC note, or somewhere else in the docs? I have added an extra note in the docstring as well.

Reply:
i'm making an issue to update the note and add this comment.

Reply:
https://github.com/LSSTDESC/CCL/issues/502